ur_robot_driver/resources/ros_control.urscript

```diff
@@ -1,20 +1,20 @@
 {{BEGIN_REPLACE}}
 
-steptime = get_steptime()
+global steptime = get_steptime()
```

Comment:
We could probably replace all occurrences of `steptime` with `get_steptime()` to prevent this global variable. I am unsure, however, whether this will have impact on the script performance.

Reply:
I had the same thought and the same doubt...